src/config/config.go

```diff
@@ -145,6 +147,29 @@ func (c *LogConfig) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// MonitorConfig tuple
+type MonitorConfig struct {
+	MonitorAddress string `json:monitor-address`
+}
+
+// DefaultMonitorConfig returns default monitor config.
+func DefaultMonitorConfig() *MonitorConfig {
+	return &MonitorConfig{
+		MonitorAddress: "0.0.0.0:13308",
```

Comment:
s/13308/18080/g

Reply:
Ha! I make a mistake here and it`s fixed just now.